src/Microsoft.Azure.WebJobs.ServiceBus/Constants.cs

```diff
@@ -23,5 +23,7 @@ public static JsonSerializerSettings JsonSerializerSettings
                 return _serializerSettings;
             }
         }
+
+        public const string DefaultConnectionStringName = "ServiceBus";
```

Comment:
The default connection string _name_ becomes `ServiceBus`. The way it would be defined is under a `ConnectionStrings` section, which in environment variables would be `ConnectionStrings:ServiceBus`, the JSON configuration, it would be under a `ConnectionString` object property.

Reply:
Do we follow this pattern for the other extensions? Aren't the storage connection strings still prefixed AzureWebJobs? What about EventHub?

Reply:
That has been a source of confusion for a while, principally in the context of Functions. I know storage tries the prefixed version first, with a fallback, but I would advocate for removing the prefix for other extensions that might be using it, for the default connection string names.